src/Basho/Riak/Link.php

```diff
@@ -72,7 +77,7 @@ public function getBucket()
 
     /**
      * Set the bucket name of this link.
-     * @param string $name - The bucket name.
+     * @param $bucket
```

Comment:
Isn't string the default? I am not sure I want to reload the branch just to change that... ha. :P